src/Condition/QueryEvaluatorTwofold.php

```diff
@@ -0,0 +1,68 @@
+<?php
+namespace Riesenia\Utility\Condition;
+
+/**
+ * This class will enable to provide configuration for two independent “prefixes”.
+ *
+ * Constructor accepts configuration array in format:
+ *     'prefix' => [
+ *         'placeholder' => [
+ *             'field' => 'field name',
+ *             'operators' => [available operators]
+ *         ]
+ *     ]
+ *
+ * @author John Diegor <john.diegor@gmail.com>
+ */
+class QueryEvaluatorTwofold extends QueryEvaluator
+{
+    /**
+     * Setup
+     *
+     * @param array configuration options
+     */
+    public function __construct(array $config)
```

Comment:
I don't see any point in overriding the constructor. Plus the `is_array` check doesn't make any sense, since theargument is defined as array.